cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/SsaInternals.qll

```diff
@@ -1031,8 +1031,8 @@ module SsaCached {
   }
 
   cached
-  Definition phiHasInputFromBlock(PhiNode phi, IRBlock bb) {
-    SsaImpl::phiHasInputFromBlock(phi, result, bb)
+  Definition phiHasInputFromBlockExt(PhiNode phi, IRBlock bb) {
+    SsaImpl::phiHasInputFromBlockExt(phi, result, bb)
```

Comment:
This is just a renaming at this point, right?

Reply:
This point is  556dc282d23e8f2fcffb096f83b2528a358b99e3

Reply:
Yeah, it shouldn't change any behavior, no